Microsoft.Toolkit.Uwp.Input.GazeControls/GazeFilePicker.xaml.cs

```diff
@@ -0,0 +1,382 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Uwp.Input.GazeInteraction;
+using Microsoft.Toolkit.Uwp.UI;
+using Windows.ApplicationModel.Resources;
+using Windows.Foundation;
+using Windows.Storage;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.Input.GazeControls
+{
+    internal enum FilePickerView
+    {
+        FileListing,
+        FilenameEntry,
+        FileOverwriteConfirmation
+    }
+
+    /// <summary>
+    /// Provides file picker dialogs optimized for gaze input
+    /// </summary>
+    public partial class GazeFilePicker : ContentDialog
```

Comment:
Can this be used directly, should it be abstract?

Reply:
Good point. Made it abstract. Thanks.